app/src/components/mainWindow.ts

```diff
@@ -64,35 +62,31 @@ function maybeInjectCss(browserWindow) {
   });
 }
 
-function clearCache(browserWindow, targetUrl = null) {
+async function clearCache(browserWindow: BrowserWindow): Promise<void> {
   const { session } = browserWindow.webContents;
-  session.clearStorageData(() => {
-    session.clearCache(() => {
-      if (targetUrl) {
```

Comment:
> Is this still needed? I vaguely remember this is to reload the webpage or something when the cache is cleared

@jiahaog from what I remember, I had to remove this because in recent Electron, the initial caller no longer gets a `targetUrl`.